README.rst

```diff
@@ -37,7 +37,18 @@ framework. This project contains no plug-ins. You are free to use:
 Prerequisites
 -------------
 
-The supported versions of Python are Python >= 3.5.
+The supported versions of Python are Python >= 3.5.  Envisage requires:
 
 * `apptools <https://github.com/enthought/apptools>`_
 * `traits <https://github.com/enthought/traits>`_
+
+Envisage has the following optional dependencies:
+
+* `Ipykernel <https://github.com/ipython/ipykernel>`_
```

Comment:
Should I link the github repo or the pypi page? Traits refers to pypi, I think pyface refers to the traits github.  In gerenal maybe pypi links would make more sense?  At the very least for non Enthought related dependencies like ipykernel

Reply:
Agreed that PyPI links make most sense. People reading the README on GitHub probably don't really care; people reading the project description on PyPI are more likely to want PyPI links.

Reply:
I went with Pypi links for non enthought stuff, and github for enthought

Reply:
ah I hadn't refreshed and just saw this comment, I will use pypi links for pyface and traitsui as well